### PR TITLE
[enhancement/additional feature] feature/custom payroll frequency bimonthly 10-25

### PR DESCRIPTION
**Pull request by dualcnhq**
- add dates for bimonthly payroll frequency but on month dates 10th of the month and 25th of the month

### CODE REVIEW

File: erpnext/hr/doctype/process_payroll/process_payroll.py [hunk 1]
```diff
@@ -366,28 +366,42 @@ def update_salary_slip_status(self, jv_name = None):
...
 				end_date = m['month_mid_end_date']
 			else:
 				start_date = m['month_mid_start_date']
 				end_date = m['month_end_date']
+
 		else:
 			start_date = m['month_start_date']
 			end_date = m['month_end_date']
 
+	if payroll_frequency == "Bimonthly(10-25)":
```
Comment (rmehta) [inline erpnext/hr/doctype/process_payroll/process_payroll.py]: there should always be a space before bracket.  `Bimonthly (10-25)`

File: erpnext/hr/doctype/process_payroll/process_payroll.py [hunk 2]
```diff
@@ -447,3 +464,39 @@ def get_month_details(year, month):
...
+
+		msd = ysd + relativedelta(months=diff_mnt) # month start date
+		nmd = msd + relativedelta(months=1) # next month date
+		month_days = cint(calendar.monthrange(cint(msd.year) ,cint(month))[1]) # days in month
+		start_date = datetime.date(msd.year, cint(month), 10) # start date of payroll for the month
+		mid_start = datetime.date(msd.year, cint(month), 26) # month mid start date
+		mid_end = datetime.date(msd.year, cint(month), 25) # month mid end date
+		med = datetime.date(nmd.year, nmd.month, 9) # month end date
+
+		print 'relativedelta next month: ', nmd
```
Comment (rmehta) [inline erpnext/hr/doctype/process_payroll/process_payroll.py]: print?

**Comment by rmehta** [comment]
@dualcnhq Thanks for your interest in contributing!

Please check my comments

[ ] can we have the dates selectable for bi-monthly?
[ ] add a test case to see if your features work
[ ] update the documentation to reflect the new changes you are proposing.

Detailed contribution guidelines here: https://github.com/frappe/erpnext/wiki/Contribution-Guidelines

**Review by rmehta — CHANGES_REQUESTED** [review]

**Comment by nabinhait** [comment]
@dualcnhq Closing the pull request for now, please reopen after fixing as per Rushabh's suggestion.

**Pull request closed without merging**